app/posts/register-trainee-teachers/2021-08-03-highest-academic-qualification.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Removing the highest academic qualification question
+description: Why we don’t think we need this question and an alternate design if we do.
+date: 2021-08-03
+---
+{% from "figure/macro.njk" import appFigure with context %}
+
+For trainees on undergraduate courses the provider has been asked for the trainee’s highest academic qualification.
+
+The question was causing users confusion. This is seems to be because it’s inconsistent with how users think about academic qualifications. A teacher training provider thinks about a trainee’s qualifications as the measure of whether a trainee is qualified and often what their ‘tariff points’ value is. In research sessions users tried to give the trainee’s actual qualifications. When users found they could only provide a single qualification most users re-read the question and were able to give the answer the service wanted.
```

Comment:
```suggestion
The question was causing users confusion. This is seems to be because it’s inconsistent with how users think about academic qualifications. A teacher training provider thinks about a trainee’s qualifications as the measure of whether a trainee is qualified for their course and often what their ‘tariff points’ value is. In research sessions users tried to give the trainee’s actual qualifications. When users found they could only provide a single qualification most users re-read the question and were able to give the answer the service wanted.
```